packages/core/src/Box/Box.spec.js

```diff
@@ -46,30 +46,25 @@ describe('Box', () => {
     expect(json).toHaveStyleRule('color', theme.colors.blue)
   })
 
-  test('bg prop sets background color', () => {
-    const json = rendererCreateWithTheme(<Box bg='green' />).toJSON()
-    expect(json).toMatchSnapshot()
-    expect(json).toHaveStyleRule('background-color', theme.colors.green)
-  })
-
   test('boxShadowSize prop sets box-shadow', () => {
     const json = rendererCreateWithTheme(<Box boxShadowSize='sm' />).toJSON()
     expect(json).toMatchSnapshot()
     expect(json).toHaveStyleRule('box-shadow', theme.boxShadows[0])
   })
 
-  test('align prop triggers warning', () => {
-    console.error = jest.fn()
-
-    rendererCreateWithTheme(<Box align='center' />).toJSON()
+  describe('deprecated prop types', () => {
+    const consoleError = console.error
+    beforeEach(() => (console.error = jest.fn()))
+    afterEach(() => (console.error = consoleError))
```

Comment:
Would you think L56 can be omitted and keep everything and simplicity inside `afterEach` function?

Reply:
Sort of same answer as my previous comment, `console.error` has to be mocked before any of the tests run to catch the output and use it for the expectations. To properly restore the function after the tests, it needs L56 to get the original `console.error` implementation unless you're thinking of a different method to do that.

Reply:
Okay so I think I understand now, I'll be refactoring these snippets to move the `consoleError = console.error` assignments into the `beforeEach` to be safer if that's along the same lines of what you're talking about, see @craigpalermo's comment.

Reply:
[Updated!](https://github.com/priceline/design-system/pull/900/files/167e8cedaa0b0eb52fcce4eded81b63f420adc8b..8452cb48afe66bbd483f8a42d0fbbc4e428d5744)